### PR TITLE
changes to order of pre-req testing, checking for plugins, etc

### PR DESCRIPTION
**Pull request by clubanderson**
after testing this utility I was able to make some changes to the pre-req checks to improve usability.  I also changed the order of the existing checks to come before the worker id query.  Without the vpc-infra or container service plugins you cannot list the storage volumes and act on Kube.  These were not being checked for in the original script. 

this script claims to "upgrade" worker nodes - "upgrade" is not a ks workers command - update and replace exists, but no upgrade.

### CODE REVIEW

**Comment by clubanderson** [comment]
added check for cancel of replace prompt.  If "N" is entered, the script stops.  I have not tested if someone enters "y" because my cluster cannot be replaced at this time.

File: px-utils/px_vpc_upgrade/px_vpc_upgrade.sh [hunk 1]
```diff
@@ -10,39 +10,16 @@ alias ic="ibmcloud"
 CLUSTER=$1
 
 [[ -z "$1" ]] && { echo "Cluster name is empty, specify a cluster name."; exit; }
-[[ -z "$2" ]] && { echo "Command name is empty. specify the command"; exit; }
+[[ -z "$2" ]] && { echo "Command name is empty, specify the command"; exit; }
 vol_ids=()
 
 if [[ "$2" == "replace" || $2 == "upgrade" ]] ; then
```
Comment (sagarkrdev) [inline px-utils/px_vpc_upgrade/px_vpc_upgrade.sh:16]: we can remove the check for upgrade here

**Review by sagarkrdev — COMMENTED** [review]
Thanks for the PR @clubanderson , some minor changes.

**Comment by clubanderson** [comment]
I have resolved changes.

**Review by sagarkrdev — APPROVED** [review]
LGTM!

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**